hack/support-bundle-collector.sh

```diff
@@ -15,16 +15,20 @@ BUNDLE_DIR="${OUTPUT_DIR}/${NODE_NAME}"
 
 mkdir -p ${BUNDLE_DIR}
 
-OS_ID=$(bash -c "source $HOST_PATH/etc/os-release && echo \$ID")
-if [ -z "$OS_ID" ]; then
-    echo "Unable to determine OS ID"
-    exit 1
-fi
-
 if [ -n "$SUPPORT_BUNDLE_COLLECTOR" ]; then
     OS_COLLECTOR="collector-$SUPPORT_BUNDLE_COLLECTOR"
 else
-    OS_COLLECTOR="collector-$OS_ID"
+    OS_RELEASE_PATH=${HOST_PATH}/etc/os-release
+
+    if [ -f ${OS_RELEASE_PATH} ]; then
+        OS_ID=$(bash -c "source ${OS_RELEASE_PATH} && echo \$ID")
+
+        if [ -z "$OS_ID" ]; then
+            echo "Unable to determine OS ID"
+        else
+            OS_COLLECTOR="collector-$OS_ID"
+        fi
+    fi
```

Comment:
```suggestion
    OS_RELEASE_PATH=${HOST_PATH}/etc/os-release

    if [ -f ${OS_RELEASE_PATH} ]; then
        OS_ID=$(bash -c "source ${OS_RELEASE_PATH} && echo \$ID")
        OS_COLLECTOR="collector-$OS_ID"
        
        if [ -z "$OS_ID" ]; then
            echo "Unable to determine OS ID"
            exit 1
        fi
    fi
```

If it's specific to OS supported here, should exit if the OS is not supported or the value is empty. However, you can tackle the empty case first.

Reply:
Got it, I've re-updated as suggested. Thank you.